wiki/History_of_osu!/2015/en.md

```diff
@@ -1,39 +1,185 @@
+9K was made rankable after being given proper use support in the editor on 30 April 2015 [[9]][r].
 
 ## May
 
-peppy releases a teaser trailer for the new game client.
+*Notice: The [forum post](https://osu.ppy.sh/community/forums/topics/334994) that announced news of the new ranking system was released on 4 June 2015, although the old system was actually fully deprecated on 1 May 2015, and full implementation of the new system was still yet to be available to the public by the end of 2015.*
+
+The old Beatmap Nominator (BN) ranking was fully deprecated on 1 May 2015, and replaced with a new, real-time ranking. This new ranking retained the original goal of the previous one, although it focused on challenge and collaboration as opposed to individual performance over a set timeframe. Public reception towards this new change was overall mostly positive, although many users did remark that the new system was confusing [[10,11]][r].
+
+Among the changes and announcements made surrounding beatmap ranking, [a change was made in the ranking process](https://osu.ppy.sh/community/forums/topics/325973) on 5 May 2015 that allowed players to dispute disqualifications made by the Quality Assurance Team (QAT), due to the lack of any way to dispute issues that the QAT may bring up. The new change made it from then on possible to submit a form if there was a dispute, where a final decision could be made regarding the map and whether it is ready to be ranked [[12]][r].
+
+The [osu!news YouTube Channel](https://www.youtube.com/channel/UCZKQIqv9O2tddMNUMAxWaqQ "YouTube") uploaded its [first video](https://www.youtube.com/watch?v=KQbudVxEjr8 "YouTube") to the public on 8 May 2015. The channel was run by various members of the community and aimed to create videos on skin showcases, tournaments, mapping, contests, and other news from the community [[12,13]][r].
 
-Links:
+The osu!keyboard (also known as the "osu! 'nono' keyboard") was added to the osu!store on 30 May 2015, retailing for about $40 at launch. Initially announced in a [tweet by peppy](https://twitter.com/ppy/status/603797988742336512) on 28 May 2015, there were only 100 keyboards available at launch, and were more akin to a keypad than a full-sized keyboard [[14,15]][r].
 
-- [Comp 3 6c](https://www.youtube.com/watch?v=lrWyqpNUwBo)
+![](img/osu-keyboard.jpg "Image of the osu! keyboard teased in the tweet by peppy")
+
+## June
+
+Implementation of the username change system in the osu!store was set live on 16 June 2015, although full functionality of the system was not available until 18 June 2015. The changes made it so that players must pay $4 USD for their first name change, $8 USD for their second, $16 USD for their third, after which the price would double with each succeeding purchase with a cap at $100 USD; however, if a user had bought an [osu!supporter tag](https://osu.ppy.sh/home/support), then their first change would be free [[16,17,18,19]][r].
```

Comment:
```suggestion
Implementation of the username change system in the osu!store was set live on 16 June 2015, although full functionality of the system was not available until 18 June 2015. The changes made it so that players had to pay $4 USD for their first name change, after which the price would double with each succeeding purchase with a cap at $100 USD. However, if a user bought an [osu!supporter tag](https://osu.ppy.sh/home/support), then their first change would be free [[16,17,18,19]][r].
```

Reply:
use of semicolon was intentional
```suggestion
Implementation of the username change system in the osu!store was set live on 16 June 2015, although full functionality of the system was not available until 18 June 2015. The changes made it so that players had to pay $4 USD for their first name change, after which the price would double with each succeeding purchase with a cap at $100 USD; however, if a user bought an [osu!supporter tag](https://osu.ppy.sh/home/support), then their first change would be free [[16,17,18,19]][r].
```